addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/ZoneThingHandler.java

```diff
@@ -100,26 +101,31 @@ public void updateChannel(ChannelUID channelUID, int state, String description)
      */
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (dscAlarmBridgeHandler == null) {
-            logger.warn("DSC Alarm bridge handler not available. Cannot handle command without bridge.");
+
+        logger.debug("handleCommand(): Command Received - {} {}.", channelUID, command);
+
+        if (command instanceof RefreshType) {
             return;
         }
 
-        if (dscAlarmBridgeHandler.isConnected()) {
-            switch (channelUID.getId()) {
-                case ZONE_BYPASS_MODE:
-                    if (command == OnOffType.OFF) {
-                        String data = String.valueOf(getPartitionNumber()) + "*1"
-                                + String.format("%02d", getZoneNumber()) + "#";
-                        dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.KeySequence, data);
-                    } else if (command == OnOffType.ON) {
-                        String data = String.valueOf(getPartitionNumber()) + "*1"
-                                + String.format("%02d", getZoneNumber()) + "#";
-                        dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.KeySequence, data);
-                    }
-                    break;
-                default:
-                    break;
+        if (dscAlarmBridgeHandler != null) {
+
+            if (dscAlarmBridgeHandler.isConnected()) {
+                switch (channelUID.getId()) {
```

Comment:
You can maybe join the `channelUID.getId() == ZONE_BYPASS_MODE` with the outer if no reason to use an if with a single case.